pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotRealtimeTableResource.java

```diff
@@ -169,19 +169,40 @@ public Map<String, String> forceCommit(
       @ApiParam(value = "Comma separated list of partition group IDs to be committed") @QueryParam("partitions")
       String partitionGroupIds,
       @ApiParam(value = "Comma separated list of consuming segments to be committed") @QueryParam("segments")
-      String consumingSegments, @Context HttpHeaders headers) {
+      String consumingSegments,
+      @ApiParam(value = "Max number of consuming segments to commit at once (default = Integer.MAX_VALUE)")
+      @QueryParam("batchSize")
+      Integer batchSize,
```

Comment:
We can directly put default in the API:
```suggestion
      @ApiParam(value = "Max number of consuming segments to commit at once (default = Integer.MAX_VALUE)")
      @QueryParam("batchSize") @DefaultValue(Integer.toString(Integer.MAX_VALUE))
      int batchSize,
```

Reply:
We can only put compile-time constant here, hence this won't work. 
Should I add `@DefaultValue("2147483647")` instead or maybe leave as it is?